cpp-client/deephaven/dhclient/CMakeLists.txt

```diff
@@ -6,8 +6,8 @@ set(CMAKE_CXX_STANDARD 17)
 # for CMAKE_INSTALL_{dir}
 include(GNUInstallDirs)
 
-find_package(Arrow REQUIRED)
-find_package(ArrowFlight REQUIRED HINTS ${Arrow_DIR})
```

Comment:
Are we sure this was not necessary for some other platform?  We also build for customers on RHEL 8 / Fedora 38, I don't remember when they were added, but git blame can probably help us figure out

Reply:
oh you mean the HINTS part. hmm

Reply:
ok re-added